tests/files/opds/book_without_license.opds

```diff
@@ -22,6 +22,7 @@
     <category term="18" scheme="http://schema.org/typicalAgeRange" label="18"/>
     <category term="http://librarysimplified.org/terms/fiction/Fiction" scheme="http://librarysimplified.org/terms/fiction/" label="Fiction"/>
     <dcterms:language>en</dcterms:language>
+    <dcterms:format>application/audiobook+json;protection="http://www.feedbooks.com/audiobooks/access-restriction"</dcterms:format>
```

Comment:
The changes to this file let me use `test_import_book_that_offers_no_license` as an end-to-end test of `extract_medium`. We're able to figure out that this book is an audiobook even though `schema:additionalType` is missing and there are no links. The only way to figure this out is to look at the `<dcterms:format>` tag.